aiogram/types/inline_keyboard.py

```diff
@@ -50,7 +50,7 @@ def add(self, *args):
             if index % self.row_width == 0:
                 self.inline_keyboard.append(row)
                 row = []
-        if len(row) > 0:
+        if row:
```

Comment:
Explicit is better than implicit

Reply:
https://google.github.io/styleguide/pyguide.html#2144-decision